react-components/src/components/Architecture/RevealButtons.tsx

```diff
@@ -64,9 +64,20 @@ export class RevealButtons {
   static KeyboardSpeed = (): ReactElement =>
     createButtonFromCommandConstructor(() => new KeyboardSpeedCommand());
 
-  static Do360Action = ({ action }: { action: Image360Action }): ReactElement =>
+  static Image360Action = ({ action }: { action: Image360Action }): ReactElement =>
```

Comment:
Having same name of `Image360Action` is confusing!
Can you update the name here?

Reply:
Renamed Image360Button and Image360Buttons